CHANGELOG.md

```diff
@@ -2,15 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## Unreleased
+## 1.0.2 (2020-06-18)
 
 ### ⚠ BREAKING CHANGES
```

Comment:
This isn't a breaking change, move this as instructions under the bug fix below

Reply:
removing section